src/applications/vaos/routes.jsx

```diff
@@ -115,7 +112,15 @@ export default function createRoutesWithStore(store) {
       </Route>
       <Route
         path="new-express-care-request"
-        component={NewExpressCareRequestLayout}
+        component={asyncLoader(() =>
+          Promise.all([
+            import(/* webpackChunkName: "express-care" */ './containers/NewExpressCareRequestLayout'),
+            import(/* webpackChunkName: "express-care" */ './reducers/expressCare'),
+          ]).then(([component, reducer]) => {
+            store.injectReducer('expressCare', reducer.default);
+            return component;
+          }),
+        )}
```

Comment:
Async load express care layout and reducer

Reply:
We already load the reducer at the app level and probably have to keep in there, since we use it to fetch the window info used on the homepage.